.changeset/empty-parrots-joke.md

```diff
@@ -0,0 +1,14 @@
+---
+"@brandingbrand/code-plugin-native-navigation": major
+"@brandingbrand/code-plugin-target-extension": major
+"@brandingbrand/code-plugin-splash-screen": major
+"@brandingbrand/code-plugin-permissions": major
+"@brandingbrand/code-plugin-app-icon": major
+"@brandingbrand/code-plugin-fastlane": major
+"@brandingbrand/code-plugin-asset": major
+"@brandingbrand/code-jest-config": major
+"@brandingbrand/code-cli-kit": major
+"@brandingbrand/code-cli": major
+---
+
+Flagship Code™ v13 rewrite
```

Comment:
this is probably ok for alpha, but we should remember to clean up change set logs for the release to have it include more detail.